go.mod

```diff
@@ -1,18 +1,35 @@
 module github.com/elastic/go-sysinfo
 
-go 1.17
+go 1.18
 
 require (
+	github.com/docker/docker v20.10.22+incompatible
 	github.com/elastic/go-windows v1.0.0
 	github.com/joeshaw/multierror v0.0.0-20140124173710-69b34d4ec901
 	github.com/prometheus/procfs v0.8.0
-	github.com/stretchr/testify v1.3.0
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
+	github.com/stretchr/testify v1.7.0
+	golang.org/x/sys v0.1.0
 	howett.net/plist v0.0.0-20181124034731-591f970eefbb
 )
 
 require (
-	github.com/davecgh/go-spew v1.1.0 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
+	github.com/Microsoft/go-winio v0.6.0 // indirect
```

Comment:
this seems off

Reply:
besides the only direct dependency added was `github.com/docker/docker`